docs/initrd.md

```diff
@@ -0,0 +1,21 @@
+# Building a custom initrd and using it in a mkosi image
+
+To build an image with a mkosi-built initrd:
```

Comment:
```suggestion
Building an image with a mkosi-built initrd is a two step process, because you will build two images - the initrd and your distribution image.
```